doc/architecture/sensor_view.adoc

```diff
@@ -0,0 +1,8 @@
+= Sensor view
+
+Sensor view imitates the input of sensors.
+Sensor-view messages are derived from ground-truth messages.
+All information regarding the environment is given with respect to the virtual sensor coordinate system, with two exceptions:
+
+* Physical technology-specific data, given with respect to the physical sensor coordinate system specified in the corresponding physical sensor’s mounting position.
```

Comment:
```suggestion
* Physical technology-specific data, e.g. image_data from CameraSensorView, given with respect to the physical sensor coordinate system specified in the corresponding physical sensor’s mounting position.
```
@max-rosin could you please check how to best add this example? (maybe some hyper links)

Reply:
@kmeids Do you like my solution?

Reply:
CCB: Solution accepted with extension in following commit.